pdi/tests/PDI_context.cxx

```diff
@@ -244,7 +244,57 @@ TEST_F(ContextTest, iterator_find)
 	}
 
 	// test case where search key is not found
-	ASSERT_FALSE(
-		this->test_context->find("desc4") != this->test_context->end()
-	); // TODO: make this an ASSERT_TRUE when operator== in Context::Iterator is available
+	ASSERT_FALSE(this->test_context->find("desc4") != this->test_context->end());
+	ASSERT_TRUE(this->test_context->find("desc5") == this->test_context->end());
```

Comment:
May need to change the assert of inequality, as previously commented.

Reply:
I prefer to keep the old assert and add the new one